doc/templates/documentation.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Refresh" content="0; url={{ pathto('index') }}" />
```

Comment:
Should probably also use a `<link rel=canonical` to tell search engines this is permanent